dragster.js

```diff
@@ -387,7 +387,7 @@
          */
         insertAfter = function (elementTarget, elementAfter) {
             if (elementTarget && elementTarget.parentNode) {
-                elementTarget.parentNode.insertBefore(elementAfter, elementTarget.nextSibling);
+                elementTarget.parentNode.insertBefore(elementAfter, finalParams.wrapDraggableElements === FALSE ? elementTarget : elementTarget.nextSibling);
```

Comment:
Do you think you can move this part of code: `finalParams.wrapDraggableElements === FALSE ? elementTarget : elementTarget.nextSibling` into a variable? It will improve code readability, IMO.